lib/widgets/gallery.dart

```diff
@@ -231,21 +232,19 @@ class _GalleryState extends State<Gallery> {
                             color: Colors.transparent,
                             child: InkWell(
                               splashColor: Asset.Colors.paleGray,
-                              onTap: () => General.instance.navigateScreen(
+                              onTap: () => navigation.navigate(
                                 context,
-                                MaterialPageRoute(
-                                  builder: (BuildContext context) =>
-                                      PhotoDetail(
-                                    photo: photo,
-                                    onPressDelete: () {
-                                      int index = _pictures.indexWhere(
-                                          (Photo compare) =>
-                                              compare.file == photo.file);
-                                      _pictures.removeAt(index);
-                                      widget.ledgerItem.picture = _pictures;
-                                      Navigator.of(context).pop();
-                                    },
-                                  ),
+                                AppRoute.photoDetail.fullPath,
```

Comment:
I think this should be ~AppRoute.photoDetail.fullPath~ => AppRoute.photoDetail.path

Please test this!